### ISSUE TITLE
Running example work flow failed

### ISSUE THREAD

**Issue by bilibilij** [CMPG/signet#2]
I have tested signet with your example workflow but failed. 
The HSS table is empty . Does my workflow right?

```
library("devtools")
library("graph")
library("RBGL")
library("igraph")
library(signet)
library(graphite)

paths <- graphite::pathways("hsapiens", "kegg")
kegg_human <- lapply(paths[1:3], graphite::pathwayGraph)
head(kegg_human)

data(daub13)
head(scores) # gene scores

HSS <- searchSubnet(kegg_human, scores)
```
![image](https://user-images.githubusercontent.com/49976278/140020826-11d62049-6ad5-4d67-be40-8b901b6063de.png)

**Comment by yjyhhxx** [comment]
I meet the same issue.
How can i slove this problem.

**Comment by tkchafin** [comment]
I know this is an old issue, but I ran into the same problem today. I submitted a [pull request](https://github.com/CMPG/signet/pull/3) with a fix. Just posting in case anyone else comes across this posted issue. 

Until merged, you could use my fork: https://github.com/tkchafin/signet

**Comment by agouy** [comment]
Thanks for the fix, merged into [release 1.8.3](https://github.com/CMPG/signet/releases/tag/1.8.3)
